pkg/console/subresource/authentication/cluster.go

```diff
@@ -0,0 +1,43 @@
+package authentication
+
+import (
+	"fmt"
+	"strings"
+
+	config "github.com/openshift/api/config/v1"
+	"github.com/openshift/console-operator/pkg/api"
+	"golang.org/x/exp/slices"
+)
+
+func GetOIDCOCLoginCommand(authConfig *config.Authentication, apiServerURL string) string {
+	clientConfig := GetOIDCClientConfig(authConfig, api.TargetNamespace, api.OCOIDCClientComponentName)
+	if clientConfig == nil {
+		return ""
+	}
+
+	extraArgs := ""
+	if len(clientConfig.ExtraScopes) > 0 {
+		for _, scope := range clientConfig.ExtraScopes {
+			extraArgs = fmt.Sprintf("%s --extra-args scope=%s", extraArgs, strings.TrimSpace(scope))
```

Comment:
This is `--extra-scopes` (e.g. `--extra-scopes=email,profile`) https://github.com/openshift/oc/blob/master/pkg/cli/login/login.go#L99

Reply:
That was my original solution. I'll change it back. Is [this doc](https://docs.google.com/document/d/1LJz7xyD5InFSNfSxiG8X2C3uBk0JQKSAMWdWZXwXEtI/edit) outdated?

Reply:
In terms of the flag names, this document would include same incorrect names. I'd recommend to check the flags from here https://github.com/openshift/oc/blob/79dc671bdaeafa74b92f14ad9f6d84e344608034/pkg/cli/login/login.go#L94-L99